webview-ui/src/locales/en/translation.json

```diff
@@ -1,4 +1,7 @@
 {
+	"appRoot": {
+		"customInstructions": "The user has changed the language, please speak English from now on."
```

Comment:
We should only be appending this to custom instructions if the language is non-english

Reply:
*pre-pending makes more sense, so they see the custom instructions update in case they have existing instructions there.